Automattic/jetpack
# JITM: new JS and CSS builder

Unblocks #21688

#### Changes proposed in this Pull Request:
- Make the package JS and CSS builder in line with other packages

#### Jetpack product discussion
p1636490736172600-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Run `jetpack build packages/jitm`, confirm that it builds well: `projects/packages/jitm/build/` directory was created and filled with JS and CSS files.
2. Connect Jetpack, confirm that the files get loaded: `jetpack-jitm/build/index.css` and `jetpack-jitm/build/index.js`.
3. Make sure that JITM functionality remains intact, the banners show up and look good.
